### ISSUE TITLE
Plot multiple results

### ISSUE THREAD

**Issue by lvieira135** [emmanuelparadis/psmcr#5]
Hello,

I ran the package normally for the populations and the species in my study and was able to generate an individual plot for each population. However, I was not able to plot all populations together. I opened the output, but could not fully understand how the plot is generated. I could not be sure which variable represents each parameter (e.g. the T, the N...). Can someone please shed some light on the subject?

Best regards,
Lucas

**Comment by emmanuelparadis** [comment]
Hi Lucas,
Currently, what you want to do is not possible directly. However, this would be clearly useful. I'll try to write some functions for this today or tomorrow.
In the meantime, I don't have a simple solution to your problem since the scales are determined automatically in `plot.psmc`. You'll need to modify the code directly.
Best,
Emmanuel

**Comment by emmanuelparadis** [comment]
I have pushed a new version (0.1-3) with a new function `lines.psmc` to draw the results from a PSMC on an existing plot. Besides, the function `plot.psmc` has new options (xlim and ylim) to let the user set the limits on both axes. These two functions also have now the option `col` to change the colours of the lines (grey and blue by default, respectively). So it's possible to do a figure in R usual way, e.g.:
```r
plot(PSMC1, xlim = ...., ylim = ...., col = .....
lines(PSMC2, col = .....
lines(PSMC3, col = .....
etc....
```
These two functions have their own options 'g', 'mutation.rate' and 'scaled", so it's even possible to plot the results from the same population with different parameters (e.g., mutation rate and/or generation time).

Don't hesitate to give feedback because I did very little testing.
Best,
Emmanuel
